assets/css/frontend/sensei.scss

```diff
@@ -912,7 +912,6 @@ section.entry span.course-lesson-progress { margin-left: 10px; }
     }
 
     &:active {
-      border-color: $color_links - #555;
```

Comment:
Did you mean to add this commit to #2892?

Reply:
Whoops, I did.